### ISSUE TITLE
Toggle Block Comment does not behave as expected

### ISSUE THREAD

**Issue by kendrahavens** [dotnet/roslyn#37580]
[Original customer feedback on developer community](https://developercommunity.visualstudio.com/idea/589032/feature-add-in-reverse-comments-key-sequence.html)

**Version Used**: 
Visual Studio 2019 version 16.3 Preview 1

**Steps to Reproduce**:

1. Have multiple lines of code, one commented some uncommented:
```csharp
Console.WriteLine("Hello World 1");
//Console.WriteLine("Hello World 2");
```
2. Toggle Block Comment/Uncomment - (Ctrl+Shift+/)

**Expected Behavior**: Toggle the comment
```csharp
//Console.WriteLine("Hello World 1");
Console.WriteLine("Hello World 2");
```
**Actual Behavior**:
```csharp
/*Console.WriteLine("Hello World 1");
//Console.WriteLine("Hello World 2");*/
```

I believe this was a diversion from the original customer ask back when we started implementing the toggle comment commands as users can already use (Ctrl K, C) to comment the selection. Toggle block comment was meant to toggle the selection.

**Comment by dibarbet** [comment]
2 general comments on this
1.  The command is specifically for toggle block comment, so it only acts on block comments.  I think it would be undesirable for this to also act on line comments, that would be a bit confusing.  So since nothing in the selection was commented in a block comment, there are no block comments to remove, so it adds a block comment.
As an aside, perhaps it would be worth discussing if a general "toggle comment" command that acts on all comments would be useful.
2.  During the design review for https://github.com/dotnet/roslyn/issues/34734 we did discuss swapping the commented areas, but we landed on just adding to the commented area.  See the "Adding single line comment(s)" section number 5.  IIRC we felt that when someone invoked toggle line comment on this selection
```cs
var i = 1;
var j = 2;
//k must be one less than the sum.
var k = i + j - 1;
```
that doing this would be a bad experience:
```cs
//var i = 1;
//var j = 2;
k must be one less than the sum.
//var k = i + j - 1;
```
The thought was that the user likely wanted the entire selection commented in this case.
